pkg/vet/api_resources_test.go

```diff
@@ -47,23 +47,33 @@ rolebindings                                   rbac.authorization.k8s.io      tr
 			wantErr: nil,
 		},
 		{
-			name: "invalid line",
+			name: "incorrect column name",
 			lines: `NAME                              SHORTNAMES   APIGROUP                       NAMESPACED   KIND
+namespaces                        ns                                          false        Namespace
+deployments                       deploy       apps                           true         Deployment
```

Comment:
`apps` is group name instead of group version. Suggest using real output from `kubectl` instead of writing it manually. Suggest having a test to catch future breaking changes from `kubectl api-resources`. The easiest test could be validating the output column names.

Reply:
real output from kubectl is 70+ lines long. I've selected a few of them that are similar to the current test data and added a few more. All test data are now real output from `kubectl api-resoureces`.

Also added e2e test that check the column names of `kubectl api-resoureces`